src/linear_algebra/cross_product.lean

```diff
@@ -114,6 +114,10 @@ begin
   fin_cases i; norm_num; ring,
 end
 
+lemma lie_lie_cross  (u v w : fin 3 → R) :
+  (u ×₃ v) ×₃ w = u ×₃ (v ×₃ w) - v ×₃ (u ×₃ w) :=
+by rw [leibniz_cross, add_sub_cancel]
```

Comment:
The proof should be just `lie_lie u v w`

Reply:
OK! I will change it this way!